score-xml/src/test/java/io/cloudslang/content/xml/actions/AddAttributeTest.java

```diff
@@ -0,0 +1,65 @@
+package io.cloudslang.content.xml.actions;
+
+import io.cloudslang.content.xml.utils.Constants;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Map;
+
+/**
+ * Created by markowis on 25/02/2016.
+ */
+public class AddAttributeTest {
+    String xml = "<?xml version=\"1.0\" encoding=\"UTF-8\" ?>\n" +
```

Comment:
maybe move this in a file in a "resources" folder and read it from there as it is duplicated across many tests